driver/index.js

```diff
@@ -1,5 +1,32 @@
-const { handlePickup, handleDelivered } = require('./handler');
-const eventPool = require('../eventPool');
+'use strict';
+
+const io = require('socket.io-client');
+const capsSocket = io.connect('http://localhost:3000/caps');
+
+capsSocket.on('connect', () => {
+  console.log('Driver connected to CAPS hub');
+  capsSocket.emit('join', '1-206-flowers');
```

Comment:
It's not causing any problems, but does the driver need to connect to a vendor's room?